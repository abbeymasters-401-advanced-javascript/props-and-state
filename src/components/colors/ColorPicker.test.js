import React from 'react';
import { shallow } from 'enzyme';
import ColorPicker from './ColorPicker';

describe('Color Picker', () => {

  it('renders color picker component', () => {
    const wrapper = shallow(<ColorPicker colors = {[]} />);
    expect(wrapper).toMatchSnapshot();
  });

});
