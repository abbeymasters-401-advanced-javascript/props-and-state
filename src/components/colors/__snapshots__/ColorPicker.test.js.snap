// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Picker renders color picker component 1`] = `<Fragment />`;
